client/my-sites/plugins/plugins-browser-list/index.jsx

```diff
@@ -26,6 +29,8 @@ const PluginsBrowserList = ( {
 	size,
 } ) => {
 	const isWide = useBreakpoint( '>1280px' );
+	const translate = useTranslate();
+	const yoastPlugin = plugins?.find( ( plugin ) => plugin.slug === 'wordpress-seo' );
```

Comment:
Once we move into the paid list this will need to be updated to use `wordpress-seo-premium` I believe

Reply:
We need to replace `wordpress-seo` to `wordpress-seo-premium` (which exists only in WPCOM) now. Also, let's make the code more generic by removing any yoast references as much as we can.
```suggestion
   const {
		data: spotlightPlugin,
		isFetched: spotlightPluginFetched,
		isFetching: spotlightPluginFetching,
	} = useWPCOMPlugin( `wordpress-seo-premium`);
```

Reply:
✅